contracts/staking/contracts/src/sys/MixinOwnable.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.5.9;
+
+import "../interfaces/IStakingEvents.sol";
+import "../immutable/MixinStorage.sol";
+
+
+contract MixinOwnable is
+    IStakingEvents,
+    MixinDeploymentConstants,
```

Comment:
This is to remove ambiguity in the contract linearization. Basically, the rule-of-thumb is to order dependencies from _most base-like_ to _most derived_. However, this can break when there are multiple levels of inheritance. So, to avoid this problem altogether, the inheritance lists are generated in full. In this case - `MixinStorage` depends on `MixinConstants` which depends on `MixinDeploymentConstants`.

Example of general problem:
```
Ex:
contract A {}
contract B {}
contract C {}
contract D is A, B {}

Suppose now that there is a contract E that will inherit from B{} and D{}
The following follows "most base-like" to "most-derived":
contract E is B, D {}
However, it unfolds to: B{}, A{}, B{}, D{}, E{} - which is wrong because B{} appears both before and after A{}.
The correct contract inheritance for E{} should be:
contract E is A{}, B{}, D{}

So, even though contract E{} does not directly depend on A{} - it still must be included in the dependency list
Because of this, we include all descendants in the output.
```